### PR TITLE
Cubitouchwhereami

### PR DESCRIPTION
**Pull request by cubitouch**
Adding a /project endpoint

### CODE REVIEW

File: tests/OmniSharp.Tests/ProjectFact.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        [Fact]
+        public void ProjectCommands()
+        {
+            var commands = GetCommands();
+            var projectName = GetProjectName();
+            var path = GetPath();
+            Assert.Equal(1, commands.Count);
+            Assert.Equal("OmniSharp", projectName);
+            Assert.Equal("project.json", path);
+        }
```
Comment (nosami) [inline tests/OmniSharp.Tests/ProjectFact.cs]: Insert a newline after this method... same fot the next 2

File: tests/OmniSharp.Tests/ProjectFact.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        private string GetProjectName(string source = "")
+        {
+            var workspace = TestHelpers.CreateSimpleWorkspace(source);
+            var aspnet5Context = GetAspNet5Context();
+            var controller = new ProjectSystemController(aspnet5Context, null, workspace);
+            var request = CreateRequest(source);
+            var response = controller.CurrentProject(request);
+            return response.AspNet5Project.Name;
+        }
+        private string GetPath(string source = "")
```
Comment (nosami) [inline tests/OmniSharp.Tests/ProjectFact.cs]: This method is almost identical to the previous one.

File: tests/OmniSharp.Tests/ProjectFact.cs [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        {
+            var workspace = TestHelpers.CreateSimpleWorkspace(source);
+            var aspnet5Context = GetAspNet5Context();
+            var controller = new ProjectSystemController(aspnet5Context, null, workspace);
+            var request = CreateRequest(source);
+            var response = controller.CurrentProject(request);
+            return response.AspNet5Project.Path;
+        }
+        private IDictionary<string, string> GetCommands(string source = "")
```
Comment (nosami) [inline tests/OmniSharp.Tests/ProjectFact.cs]: And this.

File: tests/OmniSharp.Tests/CurrentProjectFacts.cs [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+using Microsoft.CodeAnalysis.Text;
+using OmniSharp.AspNet5;
+using OmniSharp.Models;
+using Xunit;
+
+namespace OmniSharp.Tests
+{
+    public class CurrentProjectFacts
+    {
+        AspNet5Context _context;
```
Comment (davidfowl) [inline tests/OmniSharp.Tests/CurrentProjectFacts.cs]: private readonly

**Comment by nosami** [comment]
LGTM

**Comment by nosami** [comment]
Thanks for this!

**Comment by jchannon** [comment]
Good work @cubitouch I'll try and explain Git isnt always this hard in Jabbr :smile:

**Comment by cubitouch** [comment]
Hope so, was instructing anyway ! :)

**Pull request merged**